config/core/configmaps/features.yaml

```diff
@@ -44,3 +44,7 @@ data:
   # in Trigger objects with its rich filtering capabilities.
   # For more details: https://github.com/knative/eventing/issues/5204
   new-trigger-filters: "disabled"
+  
+  # ALPHA feature: The transport-encryption flag allows you to encrypt events in transit using the transport layer security (TLS) protocol.
+  # For more details: https://github.com/knative/eventing/issues/5957
```

Comment:
There is a lint issue here: https://github.com/knative/eventing/actions/runs/4617248930/jobs/8163277093?pr=6845

```
[trailing whitespace] reported by reviewdog 🐶
  transport-encryption: "disabled" 

Raw Output:
config/core/configmaps/features.yaml:49:  transport-encryption: "disabled" 
```

Reply:
ohh that whitespace stoping GH action ? i keep this thing in my mind. Thankyou so much @pierDipi for helping me.

Reply:
No worries, we have these automated checks so that they help us